src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -335,277 +335,199 @@ export function getWarningExplanation(enrollmentStatus) {
   return content;
 }
 
-// There are 7 options for the first block in the FAQ
-export function getFAQBlock1(enrollmentStatus) {
-  let content = null;
-  switch (enrollmentStatus) {
-    case HCA_ENROLLMENT_STATUSES.deceased:
-      content = null;
-      break;
+function wrapJSXInKeyedFragment(arrayOfJSX) {
+  return arrayOfJSX.map((jsx, i) => (
+    <React.Fragment key={i}>{jsx}</React.Fragment>
+  ));
+}
 
-    case HCA_ENROLLMENT_STATUSES.enrolled:
-      content = (
-        <>
-          <h4>
-            How can I change my address, income, or other information in my VA
-            health care records?
-          </h4>
-          <p>
-            To update your information, please submit a Health Benefits Update
-            Form (VA Form 10-10EZR).
-          </p>
-          <p>
-            <a href="/health-care/update-health-information/">
-              Find out how to submit VA Form 10-10EZR
-            </a>
-            .
-          </p>
-          <p>
-            Or you can update your address and other contact information in your
-            VA.gov profile. This will update your information across several VA
-            benefits and services.
-          </p>
-          <p>
-            <a href="/profile">Go to your profile to update your address</a>.
-          </p>
-        </>
-      );
-      break;
+export function getFAQContent(enrollmentStatus) {
```

Comment:
This function is huge, in terms of number of lines. This is mostly due to the fact that I'm declaring all of the potential content blocks within the body of the method. I had originally placed these blocks (such as `faqBlock1` and `faqBlockReapply3`) outside of this function, but since they are not used anywhere outside of this function I moved them to live inside of it.

Reply:
Also, this function is not tested directly, but this test (https://github.com/department-of-veterans-affairs/vets-website/blob/31d43733e9fb5292c8815a90112e803595df5301/src/applications/hca/tests/components/HCAEnrollmentStatusFAQ.unit.spec.js#L93-L102) provides 100% coverage of this new function and also ensures the four `getFAQBlock` functions I deleted didn't break anything.